cpp/arcticdb/codec/codec.hpp

```diff
@@ -1,89 +1,77 @@
+ * file licenses/BSL.txt.
  *
- * As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+ * As of the Change Date specified in that file, in accordance with the Business Source
+ * License, use of this software will be governed by the Apache License, version 2.0.
  */
 
 #pragma once
 
 #include <arcticdb/codec/core.hpp>
-#include <arcticdb/entity/types.hpp>
 #include <arcticdb/codec/encode_common.hpp>
+#include <arcticdb/codec/segment_header.hpp>
 #include <arcticdb/entity/protobufs.hpp>
 #include <arcticdb/entity/stream_descriptor.hpp>
-#include <arcticdb/codec/segment_header.hpp>
+#include <arcticdb/entity/types.hpp>
 
 namespace arcticdb {
 
 class Segment;
 class SegmentInMemory;
 
-using ShapesBlockTDT = entity::TypeDescriptorTag<entity::DataTypeTag<entity::DataType::INT64>, entity::DimensionTag<entity::Dimension::Dim0>>;
+using ShapesBlockTDT =
+    entity::TypeDescriptorTag<entity::DataTypeTag<entity::DataType::INT64>,
+                              entity::DimensionTag<entity::Dimension::Dim0>>;
 
-Segment encode_dispatch(
-    SegmentInMemory&& in_mem_seg,
-    const arcticdb::proto::encoding::VariantCodec &codec_opts,
-    EncodingVersion encoding_version);
+Segment encode_dispatch(SegmentInMemory&& in_mem_seg,
+                        const arcticdb::proto::encoding::VariantCodec& codec_opts,
+                        EncodingVersion encoding_version);
 
-Segment encode_v2(
-    SegmentInMemory&& in_mem_seg,
-    const arcticdb::proto::encoding::VariantCodec& codec_opts
-);
+Segment encode_v2(SegmentInMemory&& in_mem_seg,
+                  const arcticdb::proto::encoding::VariantCodec& codec_opts);
 
-Segment encode_v1(
-    SegmentInMemory&& in_mem_seg,
-    const arcticdb::proto::encoding::VariantCodec& codec_opts
-);
+Segment encode_v1(SegmentInMemory&& in_mem_seg,
+                  const arcticdb::proto::encoding::VariantCodec& codec_opts);
 
-void decode_v1(const Segment& segment,
-               const SegmentHeader& hdr,
-               SegmentInMemory& res,
-               const StreamDescriptor& desc,
-               bool is_decoding_incompletes = false);
+void decode_v1(const Segment& segment, const SegmentHeader& hdr, SegmentInMemory& res,
+               const StreamDescriptor& desc, bool is_decoding_incompletes = false);
 
-void decode_v2(const Segment& segment,
-               const SegmentHeader& hdr,
-               SegmentInMemory& res,
+void decode_v2(const Segment& segment, const SegmentHeader& hdr, SegmentInMemory& res,
                const StreamDescriptor& desc);
 
-SizeResult max_compressed_size_dispatch(
-    const SegmentInMemory& in_mem_seg,
-    const arcticdb::proto::encoding::VariantCodec &codec_opts,
-    EncodingVersion encoding_version);
+SizeResult
+max_compressed_size_dispatch(const SegmentInMemory& in_mem_seg,
```

Comment:
I prefer it how it was before to be honest. Putting the indent where the function name ends means all the indents are in different places.